Makefile

```diff
@@ -74,6 +75,14 @@ gen-java:
 	mkdir -p ./$(PROTO_GEN_JAVA_DIR)
 	$(foreach file,$(PROTO_FILES),$(call exec-command, $(PROTOC) --java_out=./$(PROTO_GEN_JAVA_DIR) $(file)))
 
+# Generate gRPC/Protobuf implementation for Kotlin.
+.PHONY: gen-kotlin
+gen-kotlin:
```

Comment:
Thank you! Please also add it here: https://github.com/open-telemetry/opentelemetry-proto/blob/main/.github/workflows/build-check.yaml

Reply:
Thanks for the suggestion, I have added a job to the workflow for GitHub actions 👍🏽